### DISCUSSION TITLE
Resilient Modulus

### DISCUSSION THREAD

**Discussion by m-sams-artc** [australiangeomechanics/agsformat#44] [Ideas]
Does the current ags format (with or without AU localisation) support resilient modulus lab test results? It is a test type with growing usage to determine change in modulus over a simulated cyclic loading scenario such as occurs in pavement and railway granular base and subbase materials (instead of using a CBR). Would like a way to have this in gINT, and be able to transfer data in an ags file. Should be columns for time, cycle number, confining pressure (sigma-3), vertical pressure (sigma-1), axial strain, resilient modulus, and any others deemed suitable

**Comment by SolidRockProgrammer** [comment]
Perhaps it can be forced into cyclic triaxial test groups, but best to make new groups
